docs/asynchronous-processing/handling-failed-product-recalculations.md

```diff
@@ -0,0 +1,82 @@
+# Handling failed product recalculations
+
+Products are recalculated in batches.
+Because the batch is processed always as a whole, some products may fail to be recalculated, but it's not possible to determine which products failed.
+If a batch fails, the whole batch will be retried (up to three times).
```

Comment:
It may be worth mentioning the official docs for this while stating we use the default configuration. hence the 3 retries - https://symfony.com/doc/current/messenger.html#retries-failures (the question is whether we want to link the current docs or rather the version of Symfony we currently use - I am sure you know the pros and cons for both :smile: - I would rather use the current version :innocent: )

Reply:
Added, thanks :)